plugins/svn/README.md

```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:** 
+
+This plugin adds some useful functions.
```

Comment:
Expand the description (`useful functions for regular svn users`).
Also, add instructions on how to enable the plugin. See [example](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/gb/README.md).